maven-artifact/src/test/java/org/apache/maven/artifact/versioning/ComparableVersionTest.java

```diff
@@ -101,6 +101,13 @@ private void checkVersionsEqual(String v1, String v2) {
         assertEquals(c2, c1, "expected " + v2 + ".equals( " + v1 + " )");
     }
 
+    private void checkVersionsEqualOrder(String v1, String v2) {
```

Comment:
The name of this method does not make it obvious what it does. It probably isn;t needed at all. Inlining it would likely be easier to read.

Reply:
The `checkVersionsEqualOrder` method is only used to check whether two versions are in the same order. Is there anything unreadable about this? Do you think newComparable affects the semantics of this method?

Reply:
Yes, what's unreadable is that I don't know which is greater and which is lesser. Or if it's doing something else then I am being confused by the name. Perhaps a clearer name would help, but it'ds also OK to just inline it.

Reply:
@elharo 
I have modified it, please review again.

Reply:
I see what's happening here now, but only because I've been over this a few times. Consider renaming this method to checkVersionsHaveSameOrder or checkVersionsAreOrderedTheSame